### PR TITLE
iOS tracker 0.8.0 blog post

### PR DESCRIPTION
**Pull request by mhadam**
Rough draft - still writing documentation, so will update links after I'm done.

### CODE REVIEW

**Comment by alexanderdean** [comment]
@alexanderdean to have a read

**Comment by Mandelliant** [comment]
@mhadam noted- will review

File: _posts/2018-02-19-snowplow-objective-c-tracker-0.8.0-released/index.md [hunk 1]
```diff
@@ -74,21 +75,25 @@ SPPushNotification *event = [SPPushNotification build:^(id<SPPushNotificationBui
...
 <h2><a name="xcode9">4. XCode 9 bugfixes</a></h2>
 
-To improve compatibility with XCode 9 we have updated the Tracker as follows:
+To improve compatibility with XCode 9 we have fixed a naming conflict in the SnowplowTests bundle ([issue #343][343]).
 
-* Fix a naming conflict in the SnowplowTests bundle ([#343][343])
+We also fixed a compile-time error found in the [Obj-C Client for Iglu][iglu-client], and updated a dependency of the Tracker.
 
-This fixes a compile-time error also found in the [Obj-C Client for Iglu][iglu-client], a dependency of the Tracker.
+^^ WHAT IS THE TICKET FOR THIS ONE?
```
Comment (mhadam) [inline _posts/2018-02-19-snowplow-objective-c-tracker-0.8.0-released/index.md]: For the iglu-client:
https://github.com/snowplow/iglu-objc-client/issues/7

File: _posts/2018-02-19-snowplow-objective-c-tracker-0.8.0-released/index.md [hunk 2]
```diff
@@ -74,21 +75,25 @@ SPPushNotification *event = [SPPushNotification build:^(id<SPPushNotificationBui
...
+
+It's also important to familiarize yourself with [how to view generated interfaces][swift-interfaces] - essential for finding the type signatures of the tracker methods when translated from Objective-C to Swift.
 
 <h2><a name="xcode9">4. XCode 9 bugfixes</a></h2>
 
-To improve compatibility with XCode 9 we have updated the Tracker as follows:
+To improve compatibility with XCode 9 we have fixed a naming conflict in the SnowplowTests bundle ([issue #343][343]).
 
-* Fix a naming conflict in the SnowplowTests bundle ([#343][343])
+We also fixed a compile-time error found in the [Obj-C Client for Iglu][iglu-client], and updated a dependency of the Tracker.
```
Comment (mhadam) [inline _posts/2018-02-19-snowplow-objective-c-tracker-0.8.0-released/index.md]: We also fixed a compile-time error found in both the Tracker and one of its dependencies, the [Obj-C Client for Iglu][iglu-client].

File: _posts/2018-02-19-snowplow-objective-c-tracker-0.8.0-released/index.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+
+* Adding `trackSelfDescribingEvent` method as an alias for `trackUnstructEvent` ([issue #272][272])
+* Replacing `NSGregorianCalendar` with NSCalendarIdentifierGregorian ([issue #329][329])
+* Adding preprocessor flags to disable OpenIDFA or IDFV as required ([issue #334][334])
+* Adding `identifyUser` as an alias for `setUserId` ([issue #341][341])
+* Dropping iOS 8 as a test target, and adding iOS 10 and 11 ([issue #344][344])
+
+Many thanks to [Josh Sklar][jrmsklar] for contributing on [issue #341][341]!
+
+JOSH DIDN'T CONTRIBUTE TO 341? HIS PR WAS 345, WHICH ISN'T MENTIONED IN THE ABOVE?
```
Comment (mhadam) [inline _posts/2018-02-19-snowplow-objective-c-tracker-0.8.0-released/index.md]: ah good catch, got mixed up by similar issues

File: _posts/2018-02-19-snowplow-objective-c-tracker-0.8.0-released/index.md (hunk 2 above)
Comment (alexanderdean) [reply _posts/2018-02-19-snowplow-objective-c-tracker-0.8.0-released/index.md]: What's the ticket for this fix @mhadam ?

Comment (mhadam) [reply _posts/2018-02-19-snowplow-objective-c-tracker-0.8.0-released/index.md]: https://github.com/snowplow/iglu-objc-client/issues/7

**Comment by alexanderdean** [comment]
The blog post is looking great, but I still can't get a handle on this sentence:

> We also fixed a compile-time error found in both the Tracker and one of its dependencies, the [Obj-C Client for Iglu][iglu-client].

* What commit does this sentence refer to?
* Did we bump the Iglu Obj-C Client dependency in this commit or another one? (If not, why are we mentioning Iglu Obj-C Client at all)

**Comment by mhadam** [comment]
I think it's better we drop the reference. It referred to an error that was found in both the tracker project itself and the obj-c client.

**Comment by alexanderdean** [comment]
Is there a commit for the compile-time error?

**Comment by mhadam** [comment]
Yes, this one:
https://github.com/snowplow/snowplow-objc-tracker/issues/343

**Comment by alexanderdean** [comment]
Cool, have removed that sentence as the previous sentence already mentions it.

@mhadam this looks good to go from me. Please update all dates to be ready for a tomorrow release.

**Comment by alexanderdean** [comment]
Just waiting on @Mandelliant's green tick...

**Comment by alexanderdean** [comment]
Ping @Mandelliant

**Comment by Mandelliant** [comment]
@alexanderdean sorry missed the ping- if this is no longer under embargo, I'll get it published!

**Comment by alexanderdean** [comment]
The release went out, removing the embargo.

@mhadam before/once the blog post has gone out, please update the release notes:

https://github.com/snowplow/snowplow-objc-tracker/releases/tag/0.8.0

**Pull request merged**